Caffeine logs its exceptions to stderr

For example:
```
Feb 20, 2020 12:45:04 PM com.github.benmanes.caffeine.cache.LocalAsyncCache lambda$handleCompletion$7
WARNING: Exception thrown during asynchronous load
java.util.concurrent.CompletionException: com.linecorp.centraldogma.common.QueryExecutionException: JSON path evaluation failed: ...
        at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:314)
        at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:319)
        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:645)
        at java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:506)
        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1705)
        at io.micrometer.core.instrument.internal.TimedRunnable.run(TimedRunnable.java:44)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: com.linecorp.centraldogma.common.QueryExecutionException: JSON path evaluation failed: ...
        at com.linecorp.centraldogma.internal.Jackson.extractTree(Jackson.java:294)
        at com.linecorp.centraldogma.internal.Jackson.extractTree(Jackson.java:274)
        at com.linecorp.centraldogma.common.JsonPathQuery.apply(JsonPathQuery.java:100)
        at com.linecorp.centraldogma.common.JsonPathQuery.apply(JsonPathQuery.java:35)
        at com.linecorp.centraldogma.server.storage.repository.RepositoryUtil.applyQuery(RepositoryUtil.java:142)
        at com.linecorp.centraldogma.server.storage.repository.Repository.lambda$getOrNull$4(Repository.java:213)
        at java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:642)
        ... 7 more
Caused by: com.jayway.jsonpath.PathNotFoundException: No results for path: ...
        at com.jayway.jsonpath.internal.path.EvaluationContextImpl.getValue(EvaluationContextImpl.java:133)
        at com.jayway.jsonpath.JsonPath.read(JsonPath.java:187)
        at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:164)
        at com.jayway.jsonpath.internal.JsonContext.read(JsonContext.java:169)
        at com.linecorp.centraldogma.internal.Jackson.extractTree(Jackson.java:292)
        ... 13 more
```

--- a/server/build.gradle
+++ b/server/build.gradle
@@ -49,6 +49,7 @@ dependencies {
     jmh 'org.mockito:mockito-core'
     // Logging
     optionalImplementation 'ch.qos.logback:logback-classic'
+    optionalImplementation 'org.slf4j:jul-to-slf4j'
 }
 
 clientDependencies {

--- a/server/src/main/java/com/linecorp/centraldogma/server/Main.java
+++ b/server/src/main/java/com/linecorp/centraldogma/server/Main.java
@@ -40,6 +40,21 @@ import com.linecorp.armeria.common.util.SystemInfo;
  */
 public final class Main {
 
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    static {
+        try {
+            // Install the java.util.logging to SLF4J bridge.
+            final Class<?> bridgeHandler =
+                    Class.forName("org.slf4j.bridge.SLF4JBridgeHandler", true, Main.class.getClassLoader());
+            bridgeHandler.getMethod("removeHandlersForRootLogger").invoke(null);
+            bridgeHandler.getMethod("install").invoke(null);
+            logger.debug("Installed the java.util.logging-to-SLF4J bridge.");
+        } catch (Throwable cause) {
+            logger.debug("Failed to install the java.util.logging-to-SLF4J bridge:", cause);
+        }
+    }
+
     enum State {
         NONE,
         INITIALIZED,
@@ -47,8 +62,6 @@ public final class Main {
         STOPPED,
         DESTROYED
     }
-
-    private static final Logger logger = LoggerFactory.getLogger(Main.class);
 
     private static final File DEFAULT_DATA_DIR =
             new File(System.getProperty("user.dir", ".") + File.separatorChar + "data");

